sdk/servicebus/service-bus/src/serviceBusError.ts

```diff
@@ -172,8 +172,6 @@ export function translateServiceBusError(err: AmqpError | Error): ServiceBusErro
  *
  * @param err An error to check to see if it's of type ServiceBusError
  */
-export function isServiceBusError(
-  err: Error | AmqpError | ServiceBusError
-): err is ServiceBusError {
-  return (err as Error | ServiceBusError).name === "ServiceBusError";
+export function isServiceBusError(err: unknown): err is ServiceBusError {
+  return (err as Error | ServiceBusError)?.name === "ServiceBusError";
```

Comment:
can this not be just `(err as Error)` instead of `(err as Error | ServiceBusError)`?

Reply:
Updates `err: unknown` to be `err: any` so I don't have to cast `err` at all in this line. For our purposes, `any` works fine.